packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -219,7 +219,7 @@ class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: S
     val stripeAccountId = getValOr(params, "stripeAccountId", null)
     val urlScheme = getValOr(params, "urlScheme", null)
     val setUrlSchemeOnAndroid = getBooleanOrFalse(params, "setUrlSchemeOnAndroid")
```

Comment:
I think what you really want is to add setUrlSchemeOnAndroid  in stripe.dart and get rid of the change below